modules/cnf-modifying-and-applying-the-default-profile.adoc

```diff
@@ -0,0 +1,15 @@
+// CNF-950 4.7 Modifying and applying the default profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-modifying-and-applying-the-default-profile_{context}"]
+= Modifying and applying the default profile
+
+Now you can apply the profile manually or with the toolset of your choice (for example, ArgoCD).
+
+This project contains makefile based tooling that can be used as follows (from the project root):
+
+----
+FEATURES_ENVIRONMENT=cn-ran-overlays FEATURES=ran-profile make feature-deploy
```

Comment:
Is this a command? If so, apply [source,terminal] and add a # or $ as a command prompt. If "from the project root" means it should be run as root. then you could omit that phrase and just include the #.

Reply:
Approved.